lib/tasks/argo.rake

```diff
@@ -19,6 +19,7 @@ task :default => [:ci]
 
 task :ci do
   ENV['RAILS_ENV'] = 'test'
+  WebMock.allow_net_connect!
```

Comment:
This is in `config/initializers/webmock.rb`, is it required here also?  Could it be placed in a more specific rake task to clarify the tasks that depend on it?